src/public/modules/forms/admin/controllers/admin-form.client.controller.js

```diff
@@ -152,7 +155,18 @@ function AdminFormController(
     } else {
       angular.element('body').removeClass('mobile-scroll-lock')
     }
-    $scope.collab.isModalShown = !$scope.collab.isModalShown
+    $uibModal.open({
+      animation: false,
+      templateUrl: 'modules/forms/admin/views/collaborator.client.modal.html',
+      windowClass: 'full-screen-modal-window',
+      controller: 'CollaboratorModalController',
+      scope: $scope,
```

Comment:
can we be more explicit here about which parts of the scope we are passing in?

Reply:
I think its okay to pass in the whole scope which is similar to the previous implementation

Reply:
alright. we are actually moving away from this pattern if you look at other uses of uib modal across the code base. But not the most important now that we are moving to react